app/views/candidate_mailer/application_rejected_all_applications_rejected.text.erb

```diff
@@ -20,6 +20,9 @@ Contact <%= @course.provider.name %> directly if you have any questions about th
   You’ve now had decisions about all the courses you applied for. You did not get any offers, so you can apply again.
 <% end %>
 
-Your last application has been saved. You can make changes before you submit your new application.
+If now’s the right time for you, you can apply for teacher training again this year. 
 
+Your last application has been saved. You can make changes and choose a different course before you submit a new application.
+
+Sign in to apply again:
```

Comment:
For the sake of consistency, should there be an empty line here before the magic link? 

(I'm not sure if that would get rendered in the email, so not sure if it matters.)

Reply:
Based on the screenshot above, I think the extra line does get rendered.

Reply:
Good spot. Thanks!